test_runner/regress/test_pageserver_getpage_throttle.py

```diff
@@ -0,0 +1,118 @@
+import json
+import uuid
+
+from anyio import Path
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnvBuilder, PgBin
+from fixtures.pg_version import PgVersion
+from fixtures.types import TenantId, TimelineId
+from fixtures.utils import wait_until
+
+
+def test_pageserver_getpage_throttle(neon_env_builder: NeonEnvBuilder, pg_bin: PgBin):
```

Comment:
Interesting if this passes in normal CI/debug mode -- I would have bet that it needed to be in test_runner/performance to run stably :shrug:

Reply:
will rerun it 100 times before merging